### PR TITLE
Support for hapi 9 + 13

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots @justinbeckwith PTAL.

**Comment by JustinBeckwith** [comment]
This is great, but I'm actually on version 13 o.0

**Comment by matthewloring** [comment]
Our unit tests are now passing for versions 9-13. I need to dig through the release notes for each of these versions just to ensure that there aren't new routing/handling mechanisms that need to be traced but the functionality that was in 8 is now preserved across 8-13.

File: test/hooks/test-trace-hapi.js [hunk 1]
```diff
@@ -122,6 +123,9 @@ Object.keys(versions).forEach(function(version) {
     });
 
     it('should accurately measure get time, after + get', function(done) {
+      if (version.substring(4) > 10) {
```
Comment (ofrobots) [inline test/hooks/test-trace-hapi.js]: What's the rationale for early done on hapi10?

Comment (ofrobots) [inline test/hooks/test-trace-hapi.js]: er.. > 10.

Comment (matthewloring) [inline test/hooks/test-trace-hapi.js]: https://github.com/hapijs/hapi/issues/2850 The `after` method was removed in 11.

Comment (ofrobots) [inline test/hooks/test-trace-hapi.js]: Please add a comment here with the rationale.

Comment (matthewloring) [inline test/hooks/test-trace-hapi.js]: done.

**Comment by ofrobots** [comment]
LGTM.

**Pull request merged**